polar-core/src/partial/simplify.rs

```diff
@@ -26,52 +25,65 @@ pub fn simplify_bindings(bindings: Bindings) -> Bindings {
 }
 
 pub struct Simplifier;
-
 impl Folder for Simplifier {
```

Comment:
Are these changes just refactors? I haven't read it.

Reply:
There are two substantive changes in here:

1. Doing `maybe_unwrap_operation()` at two levels -- when we have a 1-operation partial _or_ any expression in our hand -- instead of only on 1-operation partials.
2. Simplifying `Neq` (correctly / at all).